style.css

```diff
@@ -0,0 +1,140 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+* {
+  font-family: "Poppins", sans-serif;
+}
+
+header {
+  background-color: #fff;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-around;
+  width: 100%;
+  height: 48px;
+  margin-left: 6px;
+  margin-top: -7px;
+  gap: 38%;
+}
+
+.logo {
+  display: flex;
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 20px;
+  text-align: center;
+  color: #6070ff;
+  width: 76px;
```

Comment:
[Optional] I would suggest removing the fixed width values as we have to make our designs responsive, you can use percentage values if needed